package.json

```diff
@@ -20,21 +20,23 @@
   "license": "MIT",
   "dependencies": {
     "alexa-app": "alexa-js/alexa-app",
-    "alexa-verifier-middleware": "^0.1.8",
+    "alexa-verifier-middleware": "^0.1.9",
     "bluebird": "^3.4.7",
     "body-parser": "~1.16.0",
     "ejs": "~2.5.5",
     "express": "~4.14.1",
     "hotswap": "^1.1.0",
-    "lodash.defaults": "^4.2.0"
+    "lodash.defaults": "^4.2.0",
+    "lodash.endswith": "^4.2.1",
+    "lodash.startswith": "^4.2.1"
   },
   "devDependencies": {
-    "mocha": "^2.3.4",
-    "chai": "^3.4.1",
-    "supertest": "^2.0.1",
+    "mocha": "^3.2.0",
```

Comment:
I previously downgraded some dependencies here to make things work with older versions of node.